VasilzevaAA/lab_1/Main.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+int main() {
+	std::string in_str; // ������� ������
+	std::cout << "Write a sentance: ";
+	std::getline(std::cin, in_str);
+	in_str = in_str + " "; // ������ ������ ��� ��������� ������� ��������� �� �������
+
+	std::vector <std::string> word; // ������ ��� ���� �� ��������� ����

```

Comment:
Currently, you have 2 arrays that effectively contain 2 attributes of the same word: word itself and its summ_code. You can extract those properties into separate struct e.g. 
```
struct Word {
  std::string word;
  int codes_sum;
};
```
And work with std::vector<Word>